src/main/java/com/epam/controller/IndexController.java

```diff
@@ -1,19 +1,34 @@
 package com.epam.controller;
 
+import com.epam.model.User;
+import com.epam.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
 
-import java.time.LocalDateTime;
-import java.util.HashMap;
-import java.util.Map;
 
 @Controller
+@RequestMapping(path="/demo")
 public class IndexController {
-    @GetMapping("/")
-    public ModelAndView index() {
-        Map<String, String> model = new HashMap<>();
-        model.put("name", "Alexey");
-        return new ModelAndView("index", model);
+    @Autowired
+    private UserRepository userRepository;
+
+    @GetMapping(path="/add")
+    public @ResponseBody String addNewUser (@RequestParam String name) {
+
+
+        User n = new User();
```

Comment:
U shouldn't perform such kind of actions in Controllet layer. I told u, that your first step is to implement persistence layer. 
If u want implement controller layer just right now - read about architecture patterns. You should have persistence layer (repositories), Service layer and Controller layer.
The only task of controller - cal method from service, wrap result into ResponseEntity (handle exceptions maybe). This logic can't be placed here.
Btw, do u know about lombok val / var feature? I think it's pretty cool. Google it)

Reply:
And please, delete useless space lines everywhere